neqo-transport/Cargo.toml

```diff
@@ -22,6 +22,7 @@ indexmap = { version = "2.2", default-features = false } # See https://github.co
 log = { workspace = true }
 neqo-common = { path = "../neqo-common" }
 neqo-crypto = { path = "../neqo-crypto" }
+mtu = { path = "../../mtu", default-features = false }
```

Comment:
Referencing your local `mtu` repository?

In case you haven't seen it, for local development I find `cargo`'s [dependency override](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) feature handy.